src/types/onyx/Policy.ts

```diff
@@ -980,6 +1000,36 @@ type SageIntacctConnectiosConfig = OnyxCommon.OnyxValueWithOfflineFeedback<{
         userID: string;
     };
 
+    /** Sage Intacct export configs */
+    export: OnyxCommon.OnyxValueWithOfflineFeedback<{
+        /** Export date type */
+        exportDate: ValueOf<typeof CONST.SAGE_INTACCT_EXPORT_DATE>;
+
+        /** The e-mail of the exporter */
+        exporter: string;
+
+        /** Defines how non-reimbursable expenses are exported */
+        nonReimbursable: ValueOf<typeof CONST.SAGE_INTACCT_NON_REIMBURSABLE_EXPENSE_TYPE>;
+
+        /** Account that receives the non-reimbursable expenses */
+        nonReimbursableAccount: string;
+
+        /** Default vendor used for credit card transactions of non-reimbursable bill */
+        nonReimbursableCreditCardChargeDefaultVendor: string | null;
```

Comment:
I don't think we need `| null` here (or for any of the other vendors) fwiw.

Reply:
Yup, just changed that locally. Will push an update soon :)